.github/workflows/frontend-ci.yml

```diff
@@ -113,6 +113,13 @@ jobs:
       - name: Create any cache tables
         run: python manage.py createcachetable
 
+        # Populate the database with a dandiset for FileBrowser tests
+      - name: Create a superuser
+        run: python manage.py createsuperuser --noinput --email admin@admin.com
+
+      - name: Create Dandiset
+        run: python manage.py create_dev_dandiset --owner admin@admin.com
```

Comment:
```suggestion
        run: python manage.py create_dev_dandiset --owner admin@example.com
```

Reply:
fixed in 1258b94bb37a266cc6262f55b302201715c28300